src/renderer/class/Directory.js

```diff
@@ -104,7 +107,9 @@ export default class Directory {
    * @memberof Directory
    */
   selectByFile(file) {
-    const index = this.getFileIndex(file);
+    //const index = this.getFileIndex(file);
```

Comment:
I want to leave that in there for now, just because I might switch back to using that method instead of Array.indexOf. It'll be either removed or re-implemented in the next PR.